### ISSUE TITLE
Jenkins failures due to missing publish_scaladoc.sh

### ISSUE THREAD

**Issue by heuermh** [bigdatagenomics/cannoli#21]
```
...
+ '[' 2.10 == 2.10 ']'
+ <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.6.0,SCALAVER=2.10,SPARK_VERSION=1.6.1,label=centos/ws/scripts/publish-scaladoc.sh>
/tmp/hudson6502393491935054868.sh: line 26: <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.6.0,SCALAVER=2.10,SPARK_VERSION=1.6.1,label=centos/ws/scripts/publish-scaladoc.sh>: No such file or directory
Build step 'Execute shell' marked build as failure
Recording test results
Publishing Scoverage XML and HTML report...
```

**Comment by heuermh** [comment]
I was hoping #31 would fix this, but it did not.

Looks like the same error as before ... `/ws/` in the path seems odd ...
```
+ '[' 2.10 == 2.10 ']'
+ <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.6.0,SCALAVER=2.10,SPARK_VERSION=1.6.1,label=centos/ws/scripts/publish-scaladoc.sh>
/tmp/hudson4084589706657002028.sh: line 26: <https://amplab.cs.berkeley.edu/jenkins/job/cannoli/HADOOP_VERSION=2.6.0,SCALAVER=2.10,SPARK_VERSION=1.6.1,label=centos/ws/scripts/publish-scaladoc.sh>: No such file or directory
Build step 'Execute shell' marked build as failure
Recording test results
Publishing Scoverage XML and HTML report...
```

**Comment by heuermh** [comment]
@fnothaft could you look into the Jenkins configuration for this?

**Comment by heuermh** [comment]
Fixed in Jenkins as of commit 91fc32cccfb3a115b643ced6126e49f2451c2c91.
